core/sdk-core/src/main/java/software/amazon/awssdk/core/client/config/ClientOverrideConfiguration.java

```diff
@@ -422,29 +421,11 @@ default Builder retryPolicy(RetryMode retryMode) {
 
         String defaultProfileName();
 
-        /**
-         * Set the Metric publishers to be use to publish metrics for this client. This overwrites the current list of
-         * metric publishers set on the builder.
```

Comment:
Why delete the whole comment?

Reply:
Nice catch, missed this when I did the `git-revert`